porfolio.html

```diff
@@ -16,7 +16,41 @@
             <p class="text">John Doe</p>
         </div>
     </div>
- 
+    <div class="headline">
+        <div class="primary-text">Hey There. <br>
```

Comment:
Kindly use a  `<h1>` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.